dev/breeze/src/airflow_breeze/shell/enter_shell.py

```diff
@@ -206,3 +206,31 @@ def enter_shell(**kwargs):
         console.print(CHEATSHEET, style=CHEATSHEET_STYLE)
     enter_shell_params = ShellParams(**filter_out_none(**updated_kwargs))
     run_shell_with_build_image_checks(verbose, dry_run, enter_shell_params)
+
+
+def find_airflow_container(**kwargs):
+    verbose = kwargs['verbose']
+    dry_run = kwargs['dry_run']
+    updated_kwargs = synchronize_cached_params(kwargs)
```

Comment:
No need to do that - synchronize is only needed for "python", "backend", "mysql_version" etc. - all those parameters that are actually kept in cache (and I have an idea how to get rid of the synchronize method altogether - I will do it right after I merge the big PR of mine for build images.

Reply:
ok @potiuk I will skip that part